### DISCUSSION TITLE
API Requires Domain for Key

### DISCUSSION THREAD

**Discussion by HunterJuneau** [nss-evening-cohort-14/evening-client-side#239] [Help Me!] [answered]
## NAME

Hunter Juneau

## QUESTION

The API that I want to use requires me to submit my domain to register for a key. What's the best way to go about that if I want to use it for my capstone?

**Comment by drteresavasquez** [comment, marked as answer]
Create a domain in Netlify and send them the URL is probably the easiest way if that's all they want. So create your repo and deploy.
